cluster/juju/layers/kubernetes/templates/kubedns-rc.yaml

```diff
@@ -0,0 +1,114 @@
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file should be kept in sync with cluster/images/hyperkube/dns-rc.yaml
+
+# Warning: This is a file generated from the base underscore template file: skydns-rc.yaml.base
+
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: kube-dns-v18
```

Comment:
Is there a way to NOT copy the file?  This is just going to drift again.  We usually grep for things, but sometimes things get missed or forgotten.